lib/metrics.js

```diff
@@ -66,41 +67,49 @@ Metrics.prototype.init = function (opts) {
 		logger.info({ event: 'NEXT_METRICS_DISABLED', message: 'FT_GRAPHITE_APP_UUID is set to \'false\', metrics will not be sent to FT\'s internal Graphite' });
 	}
 
-
 	/**
-	* Metrix Prefix: <platform>.<application>.<instance>.
-	* E.g. <heroku>.<front-page>.<web_1_process_cluster_worker_1_EU>.
-	*/
-	const platform = this.opts.platform || ((process.env.DYNO) ? 'heroku' : 'localhost');
-
-	// "Platform" and "app" are mandatory in the metrics prefix.
-	let prefixKeys = [platform, this.opts.app];
-
-	// "Instance" is *not* mandatory in the metrics prefix.
-	if (this.opts.instance !== false) {
-
-		// e.g. "next.heroku.front-page.web_1", "next.heroku.front-page.web_2", etc.
-		let instance = this.opts.instance || (platform === 'heroku') ? process.env.DYNO.replace('.', '_') : '_';
+	 * How metric prefixes work with FT GraphiteV2
+	 *
+	 * Prefix we send to FT GraphiteV2:
+	 * 63134cae-192b-42be-90b8-c56724a3972d.web_1_process_cluster_worker_1_EU.
+	 * (<uuid>.<instance>.)
+	 *
+	 * FT GraphiteV2 uses the UUID to authenticate the request sent by the app
+	 * that is using `next-metrics`. It also replaces the UUID in the prefix
+	 * that we've sent with a unique pattern that it has associated with the UUID
+	 * when the app was registered with FT GraphiteV2, so the prefix we send
+	 * becomes:
+	 *
+	 * next.heroku.front-page.web_1_process_cluster_worker_1_EU.
+	 * (<team>.<platform>.<application>.<instance>.)
+	 */
+
+	const prefixKeys = [FT_GRAPHITE_APP_UUID];
+
+	const disableInstanceKey = (this.opts.instance === false);
+	if (!disableInstanceKey) {
+		const platformIsHeroku = !!process.env.DYNO;
+
+		// e.g. "web_1"
+		let instanceKey = this.opts.instance || (platformIsHeroku === 'heroku') ? process.env.DYNO.replace('.', '_') : '_';
 
-		// e.g. "next.heroku.front-page.web_1_process_cluster_worker_1"
 		if (process.env.NODE_APP_INSTANCE) {
-			instance += '_process_' + process.env.NODE_APP_INSTANCE;
+			// e.g. "web_1_process_cluster_worker_1"
+			instanceKey += '_process_' + process.env.NODE_APP_INSTANCE;
 		}
-
-		// e.g. "next.heroku.front-page.web_1_process_cluster_worker_1_EU"
 		if (process.env.REGION) {
-			instance += '_' + process.env.REGION;
+			// e.g. "web_1_process_cluster_worker_1_EU"
+			instanceKey += '_' + process.env.REGION;
 		}
-
-		// e.g. "next.heroku.front-page.web_1_process_cluster_worker_1_EU_canary"
 		if (process.env.FT_APP_VARIANT && process.env.FT_APP_VARIANT === 'canary') {
-			instance += '_canary';
+			// e.g. "web_1_process_cluster_worker_1_EU_canary"
+			instanceKey += '_canary';
 		}
 
-		prefixKeys.push(instance);
+		prefixKeys.push(instanceKey);
 	}
 
-	const prefix = '.' + prefixKeys.join('.') + '.';
+	const prefix = `${prefixKeys.join('.')}.`;
```

Comment:
To make it really clear that final period isn't a mistake.

```suggestion
	const prefix = `${prefixKeys.join('.')}` + '.';
```